test/Tests.Infrastructure/DocumentStoreExtensions.cs

```diff
@@ -374,14 +374,15 @@ public JsonCommandWithPayload(string url, HttpMethod method, BlittableJsonReader
                 public override HttpRequestMessage CreateRequest(JsonOperationContext ctx, ServerNode node, out string url)
                 {
                     url = $"{node.Url}/databases/{node.Database}{_url}";
+                    var clone = _payload?.Clone(ctx);
```

Comment:
Do we support concurrent cloning of blittable?

Reply:
you are correct, I will move the `CreateRequest` to be synchronous.